docs/guides/ganache-tests.md

```diff
@@ -1,16 +1,16 @@
 # Running tests with Ganache
 
-We recommend using the built-in [Hardhat Network](../hardhat-network/README.md) network to test your
+Rather than using Ganache, we recommend using the built-in [Hardhat Network](../hardhat-network/README.md) network to test your
 smart contracts, as it generates [combined JavaScript and Solidity stack traces](../hardhat-network/README.md#solidity-stack-traces),
 making debugging easier.
 
 If you still want to run your tests using Ganache, you can do it in two ways.
 
 ## Manually running Ganache
 
-You don't need to do anything especial to use Ganache if you don't want to.
+You don't need to do anything special to use Ganache if you don't want to.
```

Comment:
Any tick to remember the difference between special and especial?

Reply:
Honestly I would have said that "especial" is not even a word, so that says something, but I just searched and apparently it is. Of course there's "especially", but there's no (commonly used) adjective form of that adverb.  Interestingly, "specially" seems to exist, but is rarely (never?) used; "specifically" seems to be the commonly used equivalent of "specially."

Reply:
So the rule of thumb would be to always use `special` and `especially` and it will be fine 99% of the time?

Reply:
Yes, and I'd even add another 9 to that! (99.9%)

Also, though, don't forget about "specifically," as it's very common.